crates/bevy_tasks/src/usages.rs

```diff
@@ -1,4 +1,52 @@
-//! Several premade usage profiles
-//! Just `Compute` for now.
+//! Definitions for a few common task pools that we want. Generally the determining factor for what
+//! kind of work should go in each pool is latency requirements.
+//!
+//! For CPU-intensive work (tasks that generally spin until completion) we have a standard Compute
+//! pool and an AsyncCompute pool. Work that does not need to be completed to present the next
+//! frame should go to the AsyncCompute pool
+//!
+//! For IO-intensive work (tasks that spend very little time in a "woken" state) we have an IO
+//! task pool. The tasks here are expected to complete very quickly. Generally they should just
+//! await receiving data from somewhere (i.e. disk) and signal other systems when the data is ready
+//! for consumption. (likely via channels)
 
-pub struct Compute(());
+use super::TaskPool;
+use std::ops::Deref;
+
+/// A newtype for a task pool for CPU-intensive work that must be completed to deliver the next
+/// frame
+#[derive(Clone)]
+pub struct ComputeTaskPool(pub TaskPool);
```

Comment:
Is it necessary to make the internal `TaskPool` public like that? Users will already be able to deref to it.

Reply:
Needed to be public so that downstream code can instantiate it. (I could add a new() function but this is a little less code)